client/executor/wasmtime/src/runtime.rs

```diff
@@ -412,25 +455,32 @@ unsafe fn do_create_runtime(
 	})
 }
 
-fn instrument(blob: &mut RuntimeBlob, semantics: &Semantics) {
+fn instrument(
+	mut blob: RuntimeBlob,
+	semantics: &Semantics,
+) -> std::result::Result<RuntimeBlob, WasmError> {
+	if semantics.stack_depth_metering {
+		const STACK_DEPTH_LIMIT: u32 = 65536;
```

Comment:
I think some comment is in order explaining this value and how it relates to the 256MiB chosen for the host stack in wasmtime.